src/usr/local/emhttp/plugins/auto-unlock/README.md

```
@@ -0,0 +1,3 @@
+**Auto Unlock**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)